src/utils/parseSpecimenBody.js

```diff
@@ -4,39 +4,32 @@ const yamlOptions = {schema: CORE_SCHEMA};
 
 const defaultMapBodyToProps = (parsedBody, rawBody) => parsedBody || rawBody;
 
-const isUndefined = (d) => d === void 0;
-
-const C_DASH = '-';
-const C_NEWLINE = '\n';
-
-const split = (text) => {
-  let i = -1;
-  while (++i < text.length) {
-    if (
-      text.charAt(i) === C_DASH &&
-      text.charAt(i + 1) === C_DASH &&
-      text.charAt(i + 2) === C_DASH &&
-      text.charAt(i + 3) === C_NEWLINE
-    ) {
-      return [text.slice(0, i), text.slice(i + 4)];
-    }
-  }
-  return [text];
+const SPLITTER = '---\n';
```

Comment:
Shouldn't this be `'\n---\n`? Otherwise it will match any line which ends with `---`, even if there are characters before it.